electronicparsers/wien2k/parser.py

```diff
@@ -1053,7 +1097,40 @@ def parse_method(self):
             sec_method.k_mesh = sec_k_mesh
             sec_k_mesh.points = kpoints[0]
             sec_k_mesh.weights = kpoints[1]
-
+        if self.inc_parser.mainfile:
+            self.inc_parser.parse()
+            atoms = self.inc_parser.get('one_atom')
+            for atom_index, atom in enumerate(atoms):
+                kappas = atom.get('kappas')
```

Comment:
Could you please add link in the comments to the userguide http://www.wien2k.at/reg_user/textbooks/usersguide.pdf and also mention the specific Table 6.6: Relativistic quantum numbers ?